WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadService.java

```diff
@@ -477,6 +518,14 @@ public void onMediaUploaded(OnMediaUploaded event) {
     @SuppressWarnings("unused")
     @Subscribe(threadMode = ThreadMode.MAIN, priority = 7)
     public void onPostUploaded(OnPostUploaded event) {
+        if (event.isError()) {
+            if (event.error != null && event.error != null) {
```

Comment:
We're checking `event.error != null` twice here - but we can actually drop this entire line, since `event.isError()` above is actually a `event.error != null` check.